litmus-portal/graphql-server/pkg/cluster/handler/handler.go

```diff
@@ -277,7 +277,7 @@ func SendRequestToSubscriber(subscriberRequest clusterOps.SubscriberRequests, r
 			Namespace:    subscriberRequest.Namespace,
 			RequestType:  subscriberRequest.RequestType,
 			ExternalData: subscriberRequest.ExternalData,
-			Username:     subscriberRequest.Username,
+			UID:          subscriberRequest.Username,
```

Comment:
I thin, this should be UID
```suggestion
			UID:          subscriberRequest.UID,
```